apps/paymaster-proxy/package.json

```diff
@@ -34,6 +35,8 @@
   "dependencies": {
     "@alchemy/aa-alchemy": "^3.6.1",
     "@alchemy/aa-core": "^3.2.0",
+    "@trpc/client": "^10.45.2",
+    "@trpc/server": "^10.45.2",
```

Comment:
how come the client needs the `@trpc/server` dependency?

Reply:
actually just tested and seems to work without. removed